layouts/partials/head.html

```diff
@@ -3,7 +3,7 @@
   <meta charset="UTF-8" />
   <meta
     name="description"
-    content="{{if .IsHome}}{{$.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}"
+    content="{{if .IsHome}}{{$.Site.Data.config.description}}{{else}}{{ replaceRE ` \[!.+\][-\+]? ` `` .Summary }}{{end}}"
```

Comment:
do you think it would make sense to pipe this through the `textprocessing.html` partial at some point because `.Summary` just ignores that (including wikilinks etc)

this would involve some refactoring though and probably won't be a part of this PR

Reply:
Checked this, and you're right, it is not needed.